app/models/curiosity_card.rb

```diff
@@ -0,0 +1,13 @@
+class CuriosityCard < ApplicationRecord
+  validates :title, presence: true, length: { minimum: 7 }
+  validates :content, presence: true, length: { in: 10..200 }
+
+  belongs_to :user
+
+  has_many_attached :images
```

Comment:
Is not required for a curiosity to have at least one image attached? 🤔

Reply:
Also I think there should be [validations](https://github.com/igorkasyanchuk/active_storage_validations) on what kind of files users are uploading(i.e content-type)
Otherwise user will be able to upload rb scripts, videos, audio...